poc-07.py

```diff
@@ -65,13 +65,36 @@ def search(self):
 
     def update(self, messages):
         """Update the storage.
-
         messages: is the collection of messages we have to create, update
         or remove."""
 
-        pass #TODO
-
-
+        """
```

Comment:
Wrong comments.
* """ are for describing Objects, API. They are exported by sphinx.
* # are for comments in the code.